ui/src/components/view/ResourceLimitTab.vue

```diff
@@ -112,7 +112,7 @@ export default {
         this.formLoading = true
         this.dataResource = await this.listResourceLimits(params)
         this.dataResource.forEach(item => {
-          form[item.resourcetype] = item.max || -1
+          form[item.resourcetype] = item.max == null ? -1 : item.max 
```

Comment:
@KlausDornsbach, the lint points to this line as having a trailing space.
```suggestion
          form[item.resourcetype] = item.max == null ? -1 : item.max
```